service/src/test/java/uk/gov/hmcts/reform/fpl/service/RespondentServiceTest.java

```diff
@@ -10,20 +15,30 @@
 import uk.gov.hmcts.reform.fpl.model.RespondentSolicitor;
 import uk.gov.hmcts.reform.fpl.model.UnregisteredOrganisation;
 import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.service.time.Time;
+import uk.gov.hmcts.reform.fpl.utils.FixedTime;
 
 import java.util.List;
 import java.util.UUID;
 
 import static java.util.Collections.emptyList;
 import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.ccd.model.ChangeOrganisationApprovalStatus.APPROVED;
+import static uk.gov.hmcts.reform.ccd.model.Organisation.organisation;
 import static uk.gov.hmcts.reform.fpl.enums.YesNo.NO;
 import static uk.gov.hmcts.reform.fpl.enums.YesNo.YES;
 import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
 import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.caseRoleDynamicList;
 
+@ExtendWith(MockitoExtension.class)
 class RespondentServiceTest {
 
-    private final RespondentService service = new RespondentService();
+    @Spy
+    private Time time = new FixedTime();
+
+    @InjectMocks
+    private RespondentService service;
```

Comment:
seeing as it is only 1 parameter I don't think there is any harm in doing
```java
private RespondentService service = new RespondentService(time);
```
and removing the annotations at the same time

Reply:
but then it is not really extendable, if new dependencies are added then you need to updated constructor (which is sensitive even to order in which dependencies are declared)